lib/deploy.js

```diff
@@ -12,6 +12,7 @@ module.exports = function(opts) {
 	var token = opts.token;
 	if (!token) throw new Error('please provide a heroku access token');
 	var commit = opts.commit;
+	var processType = opts.processType;
```

Comment:
To be backwards compatible I could default this to "web". Should that be something done here or in `haikro.js`?